src/std/crypto/etc.ss

```diff
@@ -126,14 +126,12 @@
           buf)
         #f))))
 
-(def *urandom* (open-input-file "/dev/urandom"))
-
 (def (random-bytes len)
   (let (bytes (make-u8vector len))
     (random-bytes! bytes 0 len)
     bytes))
 
 (def (random-bytes! bytes (start 0) (end (u8vector-length bytes)))
-  (let (count (read-subu8vector bytes start end *urandom*))
-    (unless (eq? count (##fx- end start))
-      (error "Could not read enough random bytes" count start end))))
+  (let (result (RAND_bytes bytes start end))
```

Comment:
does this need initialization?

Reply:
From < https://www.openssl.org/docs/manmaster/man3/RAND_bytes.html >:

On all major platforms supported by OpenSSL (including the Unix-like platforms and Windows), OpenSSL is configured to automatically seed the CSPRNG on first use using the operating systems's random generator.

Reply:
Is it reasonable to replace urandom with this?
It sounds generous and might take users by surprise.